### PR TITLE
docs(ums): migrate UMS docs to observable

### PR DESCRIPTION
**Pull request by KutsenkoA**

### CODE REVIEW

File: docs/ums/README.md [hunk 1]
```diff
@@ -123,16 +131,16 @@ export UMS_TOKEN=`curl -X POST -d '{
...
 <CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
 // via alias
-const user = await ums.getUser('Elon Musk'); 
+getUser('Elon Musk').subscribe(user => console.log(user)); 
 // via email 
-const user = await ums.getUser('elon@tesla.com'); 
+await ums.getUser('elon@tesla.com').subscribe(user => console.log(user));
```
Comment (MishaJexia) [inline docs/ums/README.md:143]: Why it is a different approach with async and without?

File: docs/ums/README.md [hunk 2]
```diff
@@ -123,16 +131,16 @@ export UMS_TOKEN=`curl -X POST -d '{
...
-To fetch an user you can look at the following methods:
+To fetch a user you can look at the following methods:
 
 <CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
 // via alias
-const user = await ums.getUser('Elon Musk'); 
+getUser('Elon Musk').subscribe(user => console.log(user));
```
Comment (MishaJexia) [inline docs/ums/README.md:141]: Can we have an example of how to get value for observables? It was big problem for me. 
For example something like this: let curUser = ums.getUser('Elon Musk').subscribe()

File: docs/ums/README.md (hunk 1 above)
Comment (KutsenkoA) [reply docs/ums/README.md]: Just forgot to remove `await`, fixed

File: docs/ums/README.md (hunk 2 above)
Comment (KutsenkoA) [reply docs/ums/README.md:141]: this is an example. the only way to get value from observable is to call `subscribe()` and obtain the value inside.

**Pull request merged**